android-ktx/build.gradle

```diff
@@ -36,7 +36,7 @@ configurations.all {
 dependencies {
     api "com.mirego.trikot:trikotFoundation:$trikot_foundation_version"
     api "com.mirego.trikot:streams:$trikot_streams_version"
-    api 'com.mirego.trikot:http:1.0.7'
+    api 'com.mirego.trikot:http:1.0.9-SNAPSHOT'
```

Comment:
Just make sure not to commit the SNAPSHOT ;)

Reply:
Yes, but there is versions (using the new IR compiler) that i need that are not published yet. I will wait for them to be published before editing and merging this PR :)